store/sqlstore/store.go

```diff
@@ -148,6 +161,12 @@ func New(settings model.SqlSettings, metrics einterfaces.MetricsInterface) *SqlS
 
 	store.initConnection()
 
+	err := store.migrate(migrationsDirectionUp)
+	if err != nil {
+		mlog.Critical("Failed to apply database migrations.", mlog.Err(err))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.